.github/workflows/main.yml

```diff
@@ -0,0 +1,126 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - "2.7"
+          - "3.6"
+          - "3.7"
+          - "3.8"
+        ansible:
+          - stable-2.8
```

Comment:
Are you sure, you want to start a new collection with ansible 2.8 in mind?

Reply:
Dropped 2.8. Starting clean.